<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addinlistdialog</class>
 <widget class="QDialog" name="addinlistdialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>312</width>
    <height>205</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="simNumberLabel">
           <property name="text">
            <string>Номер SIM-карты</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="passportLabel">
           <property name="text">
            <string>Номер паспорта</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="issueDateLabel">
           <property name="text">
            <string>Дата выдачи</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="endDateLabel">
           <property name="text">
            <string>Дата окончания срока</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLineEdit" name="simNumberEdit"/>
         </item>
         <item>
          <widget class="QLineEdit" name="passportEdit"/>
         </item>
         <item>
          <widget class="QDateEdit" name="issueDateEdit"/>
         </item>
         <item>
          <widget class="QDateEdit" name="endDateEdit"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="doneButton">
         <property name="text">
          <string>Ок</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton">
         <property name="text">
          <string>Отмена</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
